mucommander-core/src/main/java/com/mucommander/ui/action/impl/OpenAction.java

```diff
@@ -35,13 +35,8 @@
 import com.mucommander.conf.MuPreferences;
 import com.mucommander.core.desktop.DesktopManager;
 import com.mucommander.job.impl.TempExecJob;
-import com.mucommander.process.ProcessRunner;
 import com.mucommander.text.Translator;
-import com.mucommander.ui.action.AbstractActionDescriptor;
-import com.mucommander.ui.action.ActionCategory;
-import com.mucommander.ui.action.ActionDescriptor;
-import com.mucommander.ui.action.ActionFactory;
-import com.mucommander.ui.action.MuAction;
+import com.mucommander.ui.action.*;

```

Comment:
please use the [formatter](https://github.com/mucommander/mucommander/blob/master/mucommander-code-format.xml) in your IDE. I don't mind how the imports are configured but when we all use the same formatter, it would reduce the number of changes on the PRs

Reply:
@ahadas my bad, fixed